### ISSUE TITLE
Why do the cocopp colors not correspond to the order of the algorithms on input?

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#1831]
Or, in other words, how can I make sure that the first algorithm name passed to `cocopp.main` gets always the first color?

**Comment by nikohansen** [comment]
Apparently algorithms get sorted at some point or another (or many). I found:
- `ppfig.get_plotting_styles`:
```python
    foreground_algorithms = [key for key in algorithms
                             if key in genericsettings.foreground_algorithm_list]
    foreground_algorithms.sort()
```
   which is called in `ppfigs`, `pprldmany` and `pptables`.
- `pprldmany.all_single_functions`:
```python
    dictFG = pp.dictAlgByFun(dict_alg)
    for fg, tempDictAlg in sorted(dictFG.items()):
```
- `pprldmany.main`:
```python
for alg in sorted(algorithms_with_data):
```

**Comment by nikohansen** [comment]
Should be fixed with commit 4a945a7.
